diff -urN primus_vk.orig/nv_vulkan_wrapper.cpp primus_vk/nv_vulkan_wrapper.cpp
--- primus_vk.orig/nv_vulkan_wrapper.cpp	2021-08-17 13:27:58.256966057 +0200
+++ primus_vk/nv_vulkan_wrapper.cpp	2021-08-17 13:31:12.198515730 +0200
@@ -12,7 +12,7 @@
 
 
 #ifndef NV_DRIVER_PATH
-#define NV_DRIVER_PATH "/usr/lib/x86_64-linux-gnu/nvidia/libGL.so.1:libGLX_nvidia.so.0"
+#define NV_DRIVER_PATH "/usr/lib64/libGL.so.1:libGLX_nvidia.so.0"
 #endif
 #ifndef NV_BUMBLEBEE_DISPLAY
 #define NV_BUMBLEBEE_DISPLAY ":8"
